x-pack/plugins/ml/server/models/job_validation/job_validation.ts

```diff
@@ -100,6 +100,12 @@ export async function validateJob(
           ...(await validateModelMemoryLimit(client, mlClient, job, duration))
         );
       }
+
+      // if datafeed has aggregation, require job config to include a valid summary_doc_field_name
+      const datafeedAggConfig = job.datafeed_config?.aggregations ?? job.datafeed_config?.aggs;
+      if (datafeedAggConfig !== undefined && !job.analysis_config?.summary_count_field_name) {
```

Comment:
in the two conditions here, one checks for `undefined` and one checks for `!`. 
Is there a reason for the difference? or could they both be `undefined` checks?

Reply:
I see that this was two ifs combined based on a previous comment.
being a fussy nitpicker i think they should be the same so not to raised questions further down the line.

Reply:
Updated here https://github.com/elastic/kibana/pull/81923/commits/f54c8e8dc5d0419538d1270cb91407de4f45bd42